plugins/database/redshift/redshift_test.go

```diff
@@ -131,32 +134,46 @@ func TestPostgreSQL_CreateUser(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 
-	usernameConfig := dbplugin.UsernameConfig{
+	usernameConfig := dbplugin.UsernameMetadata{
 		DisplayName: "test",
 		RoleName:    "test",
 	}
 
 	// Test with no configured Creation Statement
-	_, _, err = db.CreateUser(context.Background(), dbplugin.Statements{}, usernameConfig, time.Now().Add(time.Minute))
+	_, err = db.NewUser(context.Background(), dbplugin.NewUserRequest{})
```

Comment:
I recommend also checking the returned username against a regular expression to make sure it's formed as expected. Here's an example of doing that in the MSSQL tests: https://github.com/hashicorp/vault/blob/master/plugins/database/mssql/mssql_test.go#L98